src/commands/action/bonk.ts

```diff
@@ -54,7 +54,7 @@ export const command: Command = {
         const tenor: TenorSingleton = TenorSingleton.getInstance();
         const bonker: GuildMember = <GuildMember> ctx.member;
         const bonked: GuildMember = <GuildMember> ctx.options.getMember("target") ?? null;
-        const gif: string = await tenor.getGifs("anime+bonk");
+        const gif: string = await tenor.getGifs("bonk");
```

Comment:
The amine search word has been moved to the `getGifs` method of the `TenorSingleton`.